docs/website/scripts/load-docs.sh

```diff
@@ -54,6 +45,15 @@ for release in ${ALL_RELEASES}; do
         continue
     fi
 
+    # ignore if the tag if there is no corresponding OPA binary available on the GitHub Release page
```

Comment:
I feel like there's one if too many here? :)

Reply:
I'll take care of it in the rebase. thanks!